ucm2/Xiaomi/vince/HiFi.conf

```diff
@@ -0,0 +1,110 @@
+# Use case configuration for Xiaomi Redmi 5 Plus
+# TAS2557 Speaker codec not enabled yet (missing drivers).
+
+SectionVerb {
+	EnableSequence [
+		cset "name='PRI_MI2S_RX Audio Mixer MultiMedia1' 1"
+		cset "name='QUIN_MI2S_RX Audio Mixer MultiMedia3' 1"
```

Comment:
I think you can comment out or delete the QUIN rules until we don't have working codec for speaker.